src/components/search-bar.css

```diff
@@ -55,8 +55,7 @@
 
 .react-autosuggest__suggestion {
   cursor: pointer;
-  padding: 10px 20px;
-  text-align: left;
+  padding: 5px 20px;
```

Comment:
Now it looks a little too small - maybe something like 8px?

Reply:
Oof, will expand it...